packages/ember-intl/index.js

```diff
@@ -93,6 +94,7 @@ module.exports = {
       requiresTranslation,
       stripEmptyTranslations,
       wrapTranslationsWithNamespace,
+      verbose,
     } = this.configOptions;
```

Comment:
technically the user config could override this if they really wanted to see the missing translations in build output but not error for them. But we don't have to make that public

When embroider macros is implemented for config, a cleanup of options can be done then

Reply:
I agree with how you had set `verbose: false` in line 28, to still allow the possibility of an end-developer to log missing translations, but not to document it as a public API (by updating the blueprint for `config/ember-intl.js`), in case we want to drop it later.